src/screens/ProfileScreen.tsx

```diff
@@ -44,6 +46,9 @@ export const ProfileScreen = () => {
           value={userProfile.familyName}
         />
         <Field label={t('profile-email', 'Email')} value={userProfile.email} />
+        {__DEV__ && (
+          <Field label="Active Project" value={activeProject?.name} />
+        )}
```

Comment:
🔥 nice.  I'd love to see us make project display (and even selection for rare multi-project subject cases) a first-class screen (just like accounts) someday.  We setup the hook `useActiveProject` just like `useActiveAccount` for this exact reason - we just haven't made the screen for it yet.  Not suggesting we do it in this PR, soon, or anything - just wanted to signal I'm all for it whenever it's convenient or needed.